schedule/yast/yast2_ncurses_textmode.yaml

```diff
@@ -8,10 +8,11 @@ schedule:
     - console/prepare_test_data
     - console/consoletest_setup
     - console/yast2_lan
+    - console/yast2_bootloader
 conditional_schedule:
-    pre_boot_to_desktop:
-        ARCH:
-            aarch64:
-                - boot/uefi_bootmenu
-            s390x:
-                - installation/bootloader_start
+  pre_boot_to_desktop:
+    ARCH:
+      aarch64:
+        - boot/uefi_bootmenu
+      s390x:
+        - installation/bootloader_start
```

Comment:
Looks wrong, Travis thinks so too :P

Reply:
it is wrong. because the indentation is not consistent